docs/src/ToricVarieties/CohomologyClasses.md

```diff
@@ -0,0 +1,74 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@contents
+Pages = ["CohomologyClasses.md"]
+```
+
+
+# Cohomology Classes
+
+
+## Constructors
+
+### Generic constructors
+
+```@docs
+CohomologyClass(d::ToricDivisor)
+CohomologyClass(c::ToricDivisorClass)
+CohomologyClass(l::ToricLineBundle)
+```
+
+### Addition, subtraction and scalar multiplication
+
+```@docs
+Base.:+(cc1::CohomologyClass, cc2::CohomologyClass)
+Base.:-(cc1::CohomologyClass, cc2::CohomologyClass)
+Base.:*(c::fmpq, cc::CohomologyClass)
```

Comment:
Thank you for the pointer. Sadly, this requires more adjustments throughout the toric varieties, since we have such documented methods for (at least) line bundles, divisor and divisor classes. I will adjust this now.

Reply:
All adjusted.